### DISCUSSION TITLE
Making VSCode + TS + Yarn PnP + Vue SFC intellisense work?

### DISCUSSION THREAD

**Discussion by auirarrazaval** [yarnpkg/berry#6178] [Q&A]
So the issue is the following:

I am trying to migrate a project from yarn 1 to yarn 4. I would say I'm almost there. This is a Vue 3 project, that uses Single File Components ([SFC](https://vuejs.org/guide/scaling-up/sfc.html)).

I did the yarn version upgrade (with all recommended steps), installed the editor SDKs (it installed typescript and eslint sdks) and set the Typescript Server to use the workspace version.

✅  VSCode's TS server works with no issues for ts files as it is detecting PnP packages installed with yarn

![image](https://github.com/yarnpkg/berry/assets/31079536/1fd40f65-2e73-4374-87de-1215153f03ea)

✅  I can run the server with no problems, (yarn run vite). The [SFC](https://vuejs.org/guide/scaling-up/sfc.html)s seem to compile with no isses.
❎  However, in my SFCs the typescript server or intellisense just dont't seem to work as expected, as installed packages are not recognized.
![image](https://github.com/yarnpkg/berry/assets/31079536/98e68b17-1d1c-4933-b37b-aa02bd7bcf58)

For now I think i'll just turn off the pnp mode, but I would like to make sure that this is not currently supported before doing so? Or am I missing a configuration? Or am I just completely lost here? 😅 

Any help or insight is greatly appreciated!

**Comment by ShirazC** [comment]
did you ever find a solution to this?
